src/app/zap-templates/zcl/audio-output-cluster.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Audio Output</name>
+    <code>0x050b</code>
+    <define>AUDIO_OUTPUT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface for controlling the Output on a media device such as a TV.</description>
+    <attribute side="server" code="0x0000" define="AUDIO_OUTPUT_LIST" type="OCTET_STRING" length="32" writable="false" optional="false">audio output list</attribute>
```

Comment:
Similar question here: how do we deal with list-def? I assume these are lists of a specific type rather than OCTET_STRING. Is this true? If this needs changes, could we add a GH issue to track required changes to support lists?

Reply:
If i'm not mistake this [commit](https://github.com/project-chip/connectedhomeip/pull/5020/commits/f68f3402f0263ed9b1024b63d8e6bc54aa9cb3c5) should add list to the CHIP.